src/components/tools/text-tool.tsx

```diff
@@ -79,6 +82,7 @@ export default class TextToolComponent extends BaseComponent<IProps, IState> {
         this.setState({
           value: newValue
         });
+        this.prevText = textContent.text;
```

Comment:
`prevText` needs to be updated of course...without this there were 4 renders per text update, now there are 2 - one from the autorun and another from the state update I commented on above.

Reply:
Thanks!  Obvious oversight trying to get things in at the last minute.  This line is definitely needed.